library/src/main/java/com/ibm/research/drl/dpt/anonymization/constraints/DistinctLDiversity.java

```diff
@@ -103,6 +103,11 @@ public void initialize(IPVDataset dataset, List<ColumnInformation> columnInforma
         sanityChecks(dataset);
     }
 
+    @Override
```

Comment:
same here, should upgrade copyright to 2022?

Reply:
No real changes to this file